eslint-config-azuretools/README.md

```diff
@@ -2,25 +2,28 @@
 
 This package provides extensible ESLint configs used by the Azure Tools for VS Code Team.
 
-* `eslint-config-azuretools` (for production code)
-* `eslint-config-azuretools/test` (for test code)
+* `@microsoft/eslint-config-azuretools` (for production code)
+* `@microsoft/eslint-config-azuretools/test` (for test code)
 
 ## Usage
 
 1. Install this package and its peer dependencies. Peer dependencies can be listed with the following command:
 
     ```bash
-    npm info eslint-config-azuretools peerDependencies
+    npm info @microsoft/eslint-config-azuretools peerDependencies
     ```
 
 2. Depending on which config you want to use, add the following to your `.eslintrc`:
 
     ```json
-    "extends": "azuretools"
+    "extends": "@microsoft/eslint-config-azuretools"
```

Comment:
Why the change in the name here from azuretools to "...eslint-config-azuretools"?

Reply:
Per these docs: https://eslint.org/docs/developer-guide/shareable-configs#npm-scoped-modules

Without a scope in the name:

> You can omit the `eslint-config-` and it will be automatically assumed by ESLint

With a scope in the name:

> The module name can also be customized, just note that when using scoped modules it is not possible to omit the `eslint-config-` prefix. Doing so would result in package naming conflicts, and thus in resolution errors in most of cases